src/renderer/store/certificates/certificates.saga.ts

```diff
@@ -55,10 +56,13 @@ export function* createOwnCertificate(
     peerIdAddress = 'unknown'
   }
 
+  const dmPublicKey = yield* select(directMessagesSelectors.publicKey)
+
   const userData = {
     zbayNickname: action.payload,
     commonName: hiddenServices.libp2pHiddenService.onionAddress,
-    peerId: peerIdAddress
+    peerId: peerIdAddress,
+    dmPublicKey: hexStringToArrayBuffer(dmPublicKey)
```

Comment:
`hexStringToArrayBuffer` should be used inside `@zbay/identity`, here we should pass hex string